client/my-sites/media-library/filter-bar.jsx

```diff
@@ -63,6 +66,8 @@ export class MediaLibraryFilterBar extends Component {
 		const { translate } = this.props;
 
 		switch ( filter ) {
+			case 'this-post':
+				return translate( 'This Post', { comment: 'Filter label for media list' } );
```

Comment:
I think capitalize might screw up some translations where they might want a small letter for preposition or something similar. Not exactly sure about this